AgileCoffeeShop/CoffeeShop.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace AgileCoffeeShop
+{
+    public class CoffeeShop
+    {
+        #region Members
```

Comment:
Why did you chose to use regions?

Reply:
I used regions just to organize the code.